src/validation.cpp

```diff
@@ -2585,6 +2589,9 @@ class ConnectTrace {
  */
 bool CChainState::ConnectTip(BlockValidationState& state, const CChainParams& chainparams, CBlockIndex* pindexNew, const std::shared_ptr<const CBlock>& pblock, ConnectTrace& connectTrace, DisconnectedBlockTransactions &disconnectpool)
 {
+    AssertLockHeld(cs_main);
```

Comment:
nit:
```suggestion
    AssertLockHeld(::cs_main);
```

Reply:
Thanks. Will add the `::` when I have to retouch or on the next commit that touches those lines.